seaborn/axisgrid.py

```diff
@@ -957,6 +957,37 @@ def set_titles(self, template=None, row_template=None, col_template=None,
                 self.axes.flat[i].set_title(title, **kwargs)
         return self
 
+    def refline(self, *, x=None, y=None, color='.5', linestyle='--', **line_kws):
+        """Add a reference line(s) to each facet.
+
+        Parameters
+        ----------
+        x, y : numeric
+            Value(s) to draw the line(s) at.
+        color : :mod:`matplotlib color <matplotlib.colors>`
+            Specifies the color of the reference line(s).
+        linestyle : str
+            Specifies the style of the reference line(s). Defaults to dashed.
```

Comment:
Small thing, but I think it's best not to duplicate information about defaults in the parameter description. It's too easy for things to fall out of sync if the defaults change for some reason down the road.